train.py

```diff
@@ -212,7 +211,7 @@
 
 
 if config["enable_wandb"]:
```

Comment:
The model is saved as an artifact to wandb here.
It's kind of like a backup, I don't actually use it in anyway.
I don't want to save all models as above to wandb because wandb doesn't have that much space

Reply:
I see, didn't know it had space limit.  Thanks.